### DISCUSSION TITLE
Every game of the Shantae series

### DISCUSSION THREAD

**Discussion by Animatikor** [skyline-emu/skyline#1072] [Game Requests]
I have dumped every Shantae game to see it running on the emu, but none of them even started lol.
Only Risky's Revenge and Pirate's Curse wrote an error in the log files.
First one (second in the series) crashes with this error:
E8792HOS-1Signal: Bus error (PC: 0x7C82A93A9C) (SVC: SvcWaitSynchronization)
Second one (you understand what game in the series) with this:
E9137HOS-1Signal: Bus error (PC: 0x7C82A6FA9C) (SVC: SvcWaitSynchronization)

Other games have no log file at all. Probably cuz of error log level.
Can be outdated snap problem, but Celeste starts normally, without any graphical glitches. Low fps, but idc, it's playable as for me. Ah yes, 662 snap.

**Comment by Crytonics** [comment]
Can you come to discord server, there can help you with logs and games. And you have old snap 662 so most of games wont work.
